packages/theme-language-server-common/src/css/CSSLanguageService.spec.ts

```diff
@@ -0,0 +1,128 @@
+import { Mode, Translations } from '@shopify/theme-check-common';
+import { assert, beforeEach, describe, expect, it } from 'vitest';
+import { DocumentManager } from '../documents';
+import { CSSLanguageService } from './CSSLanguageService';
+import { getRequestParams, isCompletionList } from './test/test-helpers';
+
+describe('Module: CSSLanguageService', () => {
+  let cssLanguageService: CSSLanguageService;
+  let documentManager: DocumentManager;
+
+  beforeEach(async () => {
+    documentManager = new DocumentManager(
+      undefined,
+      undefined,
+      undefined,
+      async () => 'theme', // theme schema
+      async () => false, // invalid
+    );
+    cssLanguageService = new CSSLanguageService(
+      documentManager,
+      (uri: string) => Promise.resolve(uri.includes('tae') ? 'app' : 'theme'),
+    );
+
+    await cssLanguageService.setup({
+      textDocument: {
+        completion: {
+          contextSupport: true,
+          completionItem: {
+            snippetSupport: true,
+            commitCharactersSupport: true,
+            documentationFormat: ['markdown'],
+            deprecatedSupport: true,
+            preselectSupport: true,
+          },
+        },
+      },
+    });
+  });
+
+  describe('completions', () => {
+    it('should return CSS completions in a liquid file {% stylesheet %}', async () => {
+      const params = getRequestParams(
+        documentManager,
+        'sections/section.liquid',
+        `
+          {% stylesheet %}
+            .a:hov█ {
+              color: red;
+            }
+          {% endstylesheet %}
+          <div>hello world</div>
+
+        `,
+      );
+
+      const completions = await cssLanguageService.completions(params);
+      assert(isCompletionList(completions));
+      expect(completions.items).to.have.lengthOf(357);
```

Comment:
This looks to be normal when I compare it to other implementations, I asserted that the top one is the relevant item here 😅